response/page_path.go

```diff
@@ -2,13 +2,161 @@ package response
 
 import (
 	"context"
+	"net/url"
+	"regexp"
+	"strings"
 
 	"github.com/ONSdigital/log.go/v2/log"
 )
 
-func getPagePath(ctx context.Context, uri string) string {
+var resourceEndpoints = []string{"/chartconfig", "/chartimage", "/embed", "/chart", "/resource", "/generator", "/file", "/export"}
+
+var visualisationsEndpointRegexp = regexp.MustCompile(`(^/visualisations/.+?)/`)
```

Comment:
```suggestion
var visualisationsEndpointRegexp = regexp.MustCompile(`(^/visualisations/[^/]+)/`)
```
why not use this pattern (in common with other regexp's here)?